draft-ietf-quic-qpack.md

```diff
@@ -708,17 +726,17 @@ become blocked, as described in {{state-synchronization}}.
 {:#fig-size-sync title="Table State Synchronize"}
```

Comment:
The above table still uses 'Insert Count', which might confuse some readers now that we've defined this term above to mean something else.  What this instruction sends is a Insert Count Delta, or something?

Reply:
Yep, that's a good idea, though it pushes the limits of what fits in the little box :)  I wanted Insert Count Increment, but that very much doesn't fit.

Reply:
Incidentally, does this work?  https://github.com/quicwg/base-drafts/compare/insert-count...insert-count-increment?expand=1

Reply:
Insert Count Increment is much more specific than Table State Synchronize.  The description is good.  Sadly, it doesn't have the same ring to it, but I suppose clarity is more important.  I don't really like Delta, Increment is better.  We could also keep TSS and just put Increment or IC Increment or something in the little box.

Reply:
"Synchronize" is fancy word, yeah :)  But accurate >> fancy.  I'll make the proposed change.